src/nidcpower/disabled_examples/nidcpower_advanced_sequence.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/python
```

Comment:
Why is this example disabled? We shipped #957.
Did we just forget?

Reply:
Or merge error?

Reply:
I think we just forgot. Created an issue for it - https://github.com/ni/nimi-python/issues/1454.